masterlist.yaml

```diff
@@ -4356,6 +4356,33 @@ plugins:
       - crc: 0x2e6f79c3
         util: FO4Edit v3.2.1
 
+  - name: 'DWUK_LexingtonInteriors.esp'
+    url: [ 'https://www.nexusmods.com/fallout4/mods/19120' ]
+    dirty:
+      - <<: *reqManualFix
+        crc: 0xEFA1570A
+        subs: [ "Remove edit 000E081D and 0006496C from LexingtonInteriors.esp. Modify edit xx00266B's X position to -36328.718750 to solve the issue." ]
```

Comment:
I'm not sure if this will work tbh, it's easier to use the full cleaning info and a separate message for any extra information.
As for fixing the deleted navmesh, you can't just remove them or hide them in xEdit.
You'll still need to at least save the file in the creation kit to update the NAVI record.

Reply:
None of the records that are edited here are NAVI records. For more info, refer to https://github.com/loot/fallout4/issues/190

Anyways, I'll update the PR shortly with a message for the manual fixing instruction.

Reply:
Done. I also added Lexington Interiors Holotape fix, however I can't say I recommend that mod unless the user is using the default DEF_UI tags.